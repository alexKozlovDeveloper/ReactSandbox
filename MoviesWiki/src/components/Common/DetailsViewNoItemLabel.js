import React from 'react';

import styles from '../../styles/DetailsViewNoItemLabel.css';

const DetailsViewNoItemLabel = () => <div className={styles.title}>
        Select any movie...
            </div>;

export default DetailsViewNoItemLabel;
