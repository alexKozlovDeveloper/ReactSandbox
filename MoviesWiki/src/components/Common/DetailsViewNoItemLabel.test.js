import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import DetailsViewNoItemLabel from "./DetailsViewNoItemLabel";

configure({adapter: new Adapter()});

test('DetailsViewNoItemLabel rendering test', () => {
    const component = shallow(<DetailsViewNoItemLabel />);
    expect(shallowToJson(component)).toMatchSnapshot();
 });

