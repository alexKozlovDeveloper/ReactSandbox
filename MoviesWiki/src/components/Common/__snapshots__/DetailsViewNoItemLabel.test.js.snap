// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsViewNoItemLabel rendering test 1`] = `
<div
  className="title"
>
  Select any movie...
</div>
`;
